python/whylogs/api/logger/experimental/logger/actor/process_actor.py

```diff
@@ -87,6 +106,32 @@ def close(self) -> None:
         super().close()
         self._wrapper.close()
 
+    def _signal(self, messages: Sequence[SyncMessage] = [], error: Optional[Exception] = None) -> None:
+        if self._pipe_signaler is None:
+            return
+
+        for message in messages:
+            if message.sync:
+                self._pipe_signaler.signal((message.id, error, None))
+
+    # TODO
```

Comment:
What's the TODO for? Maybe implementing the timeout? We can probably just remove the timeout for now and kick this can down the road.